docs/platform_operators/external-databases.md

```diff
@@ -142,10 +205,15 @@ In the following section, an external database is created using the bitnami post
 
 1. [Install cf-for-k8s](../deploy.md)
 
-    i. Render the final K8s template to raw K8s configuration. Pass the `db-values.yaml` file as additional file to `ytt`
+    i. Configure your [`cf-values.yml`](../deploy.md#cf-values) file
+    i. Render the final K8s template to raw K8s configuration. Pass the `db-values.yml` file as additional file to `ytt`
 
     ```bash
-    ytt -f config -f /tmp/cf-values.yml -f db-values.yaml > /tmp/cf-for-k8s-rendered.yml
+    ytt -f config -f /tmp/cf-values.yml -f db-values.yml > /tmp/cf-for-k8s-rendered.yml
```

Comment:
Again to make doc maintenance easier (and maybe to improve user experience), I'd prefer using variables for ${TMP_DIR} and maybe `/tmp/cf-values.yml` and `/tmp/cf-for-k8s-rendered.yml` as well

Reply:
OK